docs/environment.yml

```diff
@@ -0,0 +1,10 @@
+name: rtd
+channels:
+  - defaults
+  - conda-forge
+dependencies:
+  - python=3.8
```

Comment:
🧐 
do we want python 3.10 here too?

Reply:
good catch, this is a leftover when I tried different config to make it work (the key was to select `mambaforge-22.9`  in the rtd.yaml